solutions/hierarchy/core-experimentation/tests/Unittest/folder.yaml

```diff
@@ -15,9 +15,9 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Folder
 metadata:
-  name: testing.unittest
+  name: tests.unittest
```

Comment:
"unittest" is singular in the name, but plural in the display name

Reply:
fixed